test/integration/ovs/ofctrl_test.go

```diff
@@ -712,6 +713,113 @@ func TestMoveTunMetadata(t *testing.T) {
 	CheckFlowExists(t, ovsCtlClient, uint8(table.GetID()), true, expectedFlows)
 }
 
+func TestFlowWithCTMatchers(t *testing.T) {
+	br := "br09"
+	err := PrepareOVSBridge(br)
+	require.Nil(t, err, fmt.Sprintf("Failed to prepare OVS bridge: %v", err))
+	defer DeleteOVSBridge(br)
+
+	bridge := newOFBridge(br)
+	table = bridge.CreateTable(2, 3, binding.TableMissActionNext)
+
+	err = bridge.Connect(maxRetry, make(chan struct{}))
+	require.Nil(t, err, "Failed to start OFService")
+	defer bridge.Disconnect()
+
+	ofctlClient := ovsctl.NewClient(br)
+	ctIpSrc := net.ParseIP("1.1.1.1")
+	ctIpDst := net.ParseIP("2.2.2.2")
+	_, ctIpSrcNet, _ := net.ParseCIDR("1.1.1.0/24")
+	_, ctIpDstNet, _ := net.ParseCIDR("2.2.2.0/24")
```

Comment:
suggestion: for brevity you can use:
```
ctIpSrc, ctIpSrcNet, _ := net.ParseCIDR("1.1.1.1/24")
ctIpDst, ctIpDstNet, _ := net.ParseCIDR("2.2.2.2/24")
```
it should do the same thing IIRC

Reply:
Got it, updated.